api/types/image/image_inspect.go

```diff
@@ -80,11 +82,15 @@ type InspectResponse struct {
 	// DockerVersion is the version of Docker that was used to build the image.
 	//
 	// Depending on how the image was created, this field may be omitted.
+	//
+	// Deprecated: this field is deprecated, and will be removed in the next release.
 	DockerVersion string `json:",omitempty"`
 
 	// Author is the name of the author that was specified when committing the
 	// image, or as specified through MAINTAINER (deprecated) in the Dockerfile.
 	// This field is omitted if not set.
+	//
+	// Deprecated: this field is deprecated, and will be removed in the next release.
 	Author string `json:",omitempty"`
```

Comment:
Moved back to draft for now; perhaps depreciating Author is a bit too aggressive, as there's an equivalent in the OCI as well; https://github.com/opencontainers/image-spec/blob/6519a62d628ec31b5da156de745b516d8850c8e3/specs-go/v1/config.go#L98

Reply:
OK; reverted this part; not deprecating Author for now (but we can do in a follow-up)